sympy/plotting/plot.py

```diff
@@ -1712,11 +1709,23 @@ def plot_parametric(*args, **kwargs):
        [0]: parametric cartesian line: (cos(u), sin(u)) for u over (-5.0, 5.0)
        [1]: parametric cartesian line: (cos(u), u) for u over (-5.0, 5.0)
 
+    Notes
+    =====
+
+    The plotting uses an adaptive algorithm which samples recursively to
+    accurately plot the plot. The adaptive algorithm uses a random point
+    near the midpoint of two points that has to be further sampled.
+    Hence the same plots can appear slightly different.
```

Comment:
Maybe "Repeating the same plot command can give slightly different results because of random sampling."